RELEASE/scripts/autoscend/auto_bedtime.ash

```diff
@@ -679,7 +679,7 @@ boolean doBedtime()
 			{
 				auto_log_info("Pulls remaining: " + pulls_remaining(), "olive");
 			}
-			if(item_amount($item[beer helmet]) == 0)
+			if(!possessEquipment($item[beer helmet]))
```

Comment:
should use possessOutfit() here. It's in auto_equipment.ash (I wrote it because mafia's outfit functions are unreliable when you're wearing one or more pieces of the outfit).